src/scenes/VapaaSivistystyo/components/Koulutusala.js

```diff
@@ -0,0 +1,102 @@
+import _ from 'lodash'
+import React, { Component } from 'react'
+import styled from 'styled-components'
+
+import arrow from '../../../static/images/koulutusala-arrow.svg'
+import { COLORS } from "../../../modules/styles"
+import Tutkinto from './Tutkinto'
+
+const Wrapper = styled.div`
+  margin: 4px 0;
+  background-color: ${COLORS.BG_GRAY};
+  max-width: 625px;
+`
+
+const Heading = styled.div`
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  padding:  10px 20px;
+  
+  &:hover {
+    background-color: ${COLORS.BG_DARKER_GRAY};
+  }
+`
+
+const Arrow = styled.img`
+  margin-right: 20px;
+  ${props => props.rotated ? `transform: rotate(90deg);` : null}
+`
+
+const Span = styled.span`
+  margin-right: 15px;
+`
+
+const KoulutusalaList = styled.div`
+  padding:  5px 20px 10px;
+`
+
+const SubAlaWrapper = styled.div`
+  margin: 5px 0 20px;
+  font-size: 15px;
+  font-weight: bold;
```

Comment:
Pixel values should be replaced with ems or rems (TailwindCSS has sufficient style classes out of the box). These "extra" elements are not needed. TailwindCSS has style classes for all of these definitions out of the box.

Reply:
Koulutusala will be removed and this will be thus fixed.